mirgecom/initializers.py

```diff
@@ -214,7 +308,8 @@ def __call__(self, t, x_vec, eos=IdealSingleGas()):
 
 
 class Lump:
-    r"""Implement an N-dimensional Gaussian lump of mass.
+    r"""
+    Create an N-dimensional Gaussian lump of mass.
```

Comment:
Class documentation should describe nouns, not actions. E.g. "An initializer consisting of..."

(Here and elsewhere.)